### PR TITLE
ESP32: add attributes IRAM

### PR DESCRIPTION
**Pull request by uzi18**

### CODE REVIEW

File: src/platform/OneWireNg_ArduinoESP32.h [hunk 1]
```diff
@@ -161,7 +161,7 @@ class OneWireNg_ArduinoESP32: public OneWireNg_BitBang
     }
 #endif
 
-    void initDtaGpio(unsigned pin, bool pullUp)
+    void IRAM_ATTR initDtaGpio(unsigned pin, bool pullUp)
```
Comment (pstolarz) [inline src/platform/OneWireNg_ArduinoESP32.h:164]: It's not required. `initDtaGpio()` is not time critical routine.

Comment (pstolarz) [inline src/platform/OneWireNg_ArduinoESP32.h:164]: It's not required. `initDtaGpio()` is not time critical routine.

File: src/platform/OneWireNg_ArduinoESP32.h [hunk 2]
```diff
@@ -188,7 +188,7 @@ class OneWireNg_ArduinoESP32: public OneWireNg_BitBang
     }
 
 #ifdef CONFIG_PWR_CTRL_ENABLED
-    void initPwrCtrlGpio(unsigned pin)
+    void IRAM_ATTR initPwrCtrlGpio(unsigned pin)
```
Comment (pstolarz) [inline src/platform/OneWireNg_ArduinoESP32.h:191]: As above. Let's save IRAM.

Comment (pstolarz) [inline src/platform/OneWireNg_ArduinoESP32.h:191]: As above. Let's save IRAM.

**Comment by uzi18** [comment]
here you are :)

**Comment by pstolarz** [comment]
Have you checked the change on your setups (ESP32, ESP8266)? Basically I'm not opposite to this change, although don't expect drastic timing improvements (`OneWireNg_BitBang::reset()` and `OneWireNg_BitBang::touchBit()` are already IRAMed for both ESPs). I'll need to check this change with OD mode on my setups before a marge.

**Comment by pstolarz** [comment]
Let it go, I'll check it later...

**Comment by uzi18** [comment]
No but I'm sure about it is absolutely must and some OneWire forks exists only because of this change

**Comment by pstolarz** [comment]
I haven't got any issues of the lib w/o this attribute. As I said `reset()` and `touchBit()` are already IRAMed. But ok, I merged it and test it probably in the next weekend (especially for the over-drive mode).

**Pull request merged**